static/app/views/starfish/views/appStartup/screenSummary/appStartBreakdownWidget.tsx

```diff
@@ -0,0 +1,167 @@
+import styled from '@emotion/styled';
+
+import {getInterval} from 'sentry/components/charts/utils';
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
+import {OpsDot} from 'sentry/components/events/opsBreakdown';
+import LoadingContainer from 'sentry/components/loading/loadingContainer';
+import TextOverflow from 'sentry/components/textOverflow';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {formatVersion} from 'sentry/utils/formatters';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {prepareQueryForLandingPage} from 'sentry/views/performance/data';
+import MiniChartPanel from 'sentry/views/starfish/components/miniChartPanel';
+import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
+import {formatVersionAndCenterTruncate} from 'sentry/views/starfish/utils/centerTruncate';
+import {STARFISH_CHART_INTERVAL_FIDELITY} from 'sentry/views/starfish/utils/constants';
+import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
+import AppStartBreakdown, {
+  COLD_START_COLOR,
+  WARM_START_COLOR,
+} from 'sentry/views/starfish/views/appStartup/appStartBreakdown';
+import {useTableQuery} from 'sentry/views/starfish/views/screens/screensTable';
+
+function AppStartBreakdownWidget({additionalFilters}) {
+  const pageFilter = usePageFilters();
+  const location = useLocation();
+  const {query: locationQuery} = location;
+
+  const {
+    primaryRelease,
+    secondaryRelease,
+    isLoading: isReleasesLoading,
+  } = useReleaseSelection();
+
+  const query = new MutableSearch([
+    'span.op:[app.start.warm,app.start.cold]',
```

Comment:
hmm, this will include the nested app start spans too ([example](https://demo.dev.getsentry.net:7999/region/us/api/0/organizations/demo/events/?dataset=spansMetrics&field=span.description&field=release&field=span.op&field=count()&per_page=25&project=6249899&query=span.op%3A[app.start.warm%2Capp.start.cold]%20transaction%3AEmpowerPlantViewController%20release%3A[sentrydemos.ios.EmpowerPlant%400.0.27%2B1%2Csentrydemos.ios.EmpowerPlant%400.0.26%2B1]&referrer=api.starfish.mobile-startup-breakdown&statsPeriod=30d&utc=true)) we should query for just the top level app start spans for this breakdown